addons/udes_stock/__manifest__.py

```diff
@@ -32,6 +32,7 @@
         "report/external_layout_standard.xml",
         "wizard/stock_backorder_confirmation_views.xml",
         "wizard/stock_immediate_transfer_views.xml",
+        "views/assets.xml",
```

Comment:
Nit: think is forgotten to add the views for stock inventory and location in manifest if i am not missing smth